Index: 9. 알고리즘 유형별 기출문제/b. 구현/07. 럭키 스트레이트.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/9. 알고리즘 유형별 기출문제/b. 구현/07. 럭키 스트레이트.py b/9. 알고리즘 유형별 기출문제/b. 구현/07. 럭키 스트레이트.py
new file mode 100644
--- /dev/null	(date 1746523472340)
+++ b/9. 알고리즘 유형별 기출문제/b. 구현/07. 럭키 스트레이트.py	(date 1746523472340)
@@ -0,0 +1,17 @@
+# N: 현재 캐릭터의 점수
+n = input()
+mid = len(n) // 2
+
+left_sum = 0
+right_sum = 0
+
+for i in range(len(n)):
+    if i < mid:
+        left_sum += int(n[i])
+    else:
+        right_sum += int(n[i])
+
+if left_sum == right_sum:
+    print('LUCKY')
+else:
+    print('READY')
\ No newline at end of file
diff --git a/9. 알고리즘 유형별 기출문제/b. 구현/12. 기둥과 보 설치.py b/9. 알고리즘 유형별 기출문제/b. 구현/12. 기둥과 보 설치.py
new file mode 100644
diff --git a/9. 알고리즘 유형별 기출문제/b. 구현/13. 치킨 배달.py b/9. 알고리즘 유형별 기출문제/b. 구현/13. 치킨 배달.py
new file mode 100644
diff --git a/9. 알고리즘 유형별 기출문제/b. 구현/14. 외벽 점검.py b/9. 알고리즘 유형별 기출문제/b. 구현/14. 외벽 점검.py
new file mode 100644
diff --git a/9. 알고리즘 유형별 기출문제/b. 구현/08. 문자열 재정렬.py b/9. 알고리즘 유형별 기출문제/b. 구현/08. 문자열 재정렬.py
new file mode 100644
diff --git a/9. 알고리즘 유형별 기출문제/b. 구현/09. 문자열 압축.py b/9. 알고리즘 유형별 기출문제/b. 구현/09. 문자열 압축.py
new file mode 100644
diff --git a/9. 알고리즘 유형별 기출문제/b. 구현/10. 자물쇠와 열쇠.py b/9. 알고리즘 유형별 기출문제/b. 구현/10. 자물쇠와 열쇠.py
new file mode 100644
diff --git a/9. 알고리즘 유형별 기출문제/b. 구현/11. 뱀.py b/9. 알고리즘 유형별 기출문제/b. 구현/11. 뱀.py
new file mode 100644
